qa/rpc-tests/test_framework/util.py

```diff
@@ -99,11 +108,31 @@ def sync_mempools(rpc_connections, wait=1):
 
 bitcoind_processes = {}
 
-def initialize_datadir(dirname, n):
+def initialize_datadir(dirname, n,bitcoinConfDict=None,wallet=None):
     datadir = os.path.join(dirname, "node"+str(n))
     if not os.path.isdir(datadir):
         os.makedirs(datadir)
+    
+    defaults = {"server":1, "discover":0, "regtest":1,"rpcuser":"rt","rpcpassword":"rt",
```

Comment:
thanks, and good catch.  The original code hard-coded the bitcoin.conf file contents within the framework.  I realised that tests might need custom config (I do), so I added the whole config parameters.  But I missed moving this parameter out to the new test-specific config.